adding subplot to wizard

chris263 opened the pull request:
- Addin subplots to wizard
- Accept lists with tissue samples in wizard

Fixes: #5520
Fixes: #5599

lukasmueller commented:
putting a list of type tissue_samples still seems to return that error, list type not supported

chris263 commented:
1. I found an additional issue, which has now been fixed: the Trial Details page was creating the tissue samples list under the wrong type.
2. A typo (`tissue_sample` vs. `tissue_samples`) was causing the problem in the search wizard.
3. It is now possible to load a list with the `tissue_samples` type.

lukasmueller commented:
I created a subplot and a list of type subplots. When feeding the subplot list, I get again the error : "List is not of an appropriate type." For lists of type tissue_sample, this error no longer appears :-)

chris263 commented:
@lukasmueller subplots lists are validated now!

chris263 commented:
I've moved the db folder to 00198 to avoid any problems with new db patches.
Fixed subplot search.
<img width="1029" height="616" alt="Screenshot from 2025-09-15 10-11-46" src="https://github.com/user-attachments/assets/737298e3-4ab0-493b-b270-9f7d298423a9" />

alockrow commented:
I'm not getting that error anymore!
